src/Core/src/Platform/iOS/MauiView.cs

```diff
@@ -26,9 +28,28 @@ public IView? View
 
 		bool RespondsToSafeArea()
 		{
+			if (View is not ISafeAreaView sav || sav.IgnoreSafeArea)
+			{
+				return false;
+			}
+
+			if (Window is not null)
+			{
+				// If I'm inside a scrollable container then I don't need to adjust for the safe area
+				// The scroll view will handle insetting the content if needed
+				_insideScrollView ??= Superview.GetParentOfType<UIScrollView>() is not null;				
+				if (_insideScrollView.Value)
```

Comment:
why `_insideScrollView` is nullable?

Reply:
Simply to be able to initialize it once with `=??`, given that both `false` and `true` are meaningful values